include/zephyr/bluetooth/services/ias.h

```diff
@@ -17,10 +17,18 @@
  * as a part of ongoing development.
  */
 
+#include <bluetooth/conn.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+enum bt_ias_alert_lvl {
+	BT_IAS_ALERT_LVL_NO_ALERT,
+	BT_IAS_ALERT_LVL_MILD_ALERT,
+	BT_IAS_ALERT_LVL_HIGH_ALERT,
+} __packed;
```

Comment:
As you define separate functions for setting alert level (`bt_ias_client_alert_high`, `bt_ias_client_alert_mild`, `bt_ias_client_alert_stop`) I guess `bt_ias_alert_lvl` can be moved to `ias.c` as it's unused in API

Reply:
This is used in ias.c and also in ias_client.c should I then create a structure within each of those files?

Reply:
The right way of handling that is to create something like an `ias_internal.h` header file and put it there if it is shared between the `ias.c` and `ias_client.c` :)

Reply:
Done